### ISSUE TITLE
[Visual requirements - ADF - Azure data factory] Outlines have insufficient contrast ratio on Azure data factory window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3830]
**User Experience:**
Visual impaired user will find difficulty to see the non text control if that non-text control have insufficient color contrast ratio as user won't get the actual layout of screen. 

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it  

**Repro Steps:​**

1. Launch Azure Storage Explorer app.
2. Navigate and select Add data factory control at left panel.
3. Now, navigate to various edit field like Account, Subscription etc. on Add data factory window.
4. Verify contrast ratio of edit field outlines.

**Actual Result:**
Outlines have insufficient contrast ratio of 1.4:1 on Azure data factory window and due to this, visual impaired user will find difficulty to see such outlines.

**Expected Result:**
Outlines must have sufficient contrast ratio of 3:1 on Azure data factory window so that visual impaired user could easily such outlines.

![MAS1 4 11_ADF](https://user-images.githubusercontent.com/72871089/101167057-545b6180-365f-11eb-8632-e64bcd9097ce.PNG)
